manifests/params.pp

```diff
@@ -15,6 +13,17 @@
   $runinterval         = '1800'
   $runmode             = 'service'
 
+  case $::osfamily {
+    'Suse': {
+      $user            = 'root'
```

Comment:
Yes, that's the same as the AIO packages then. I don't think $user has any other use, but the other AIO and Windows logic is on $dir_owner rather than $user. It should be kept in one place.